src/pages/index.js

```diff
@@ -164,6 +203,52 @@ function handleImageClick(cardData) {
   popupImage.open(cardData);
 }
 
-//UserInfo.js
+/* -------------------------------------------------------------------------- */
+/*                                 Card Likes                                 */
+/* -------------------------------------------------------------------------- */
+
+function handleLikeClick(card) {
+  if (card.isLiked()) {
+    api
+      .dislikeCard(card.id)
+      .then(() => {
+        card.setCardLikes(false);
+      })
+      .catch((err) => console.log(err));
+  } else {
+    api
+      .likeCard(card.id)
+      .then(() => {
+        card.setCardLikes(true);
+      })
+      .catch((err) => console.log(err));
```

Comment:
Good job that you `catch` possible errors at the end of server requests. You can simplify it with `.catch(console.error);` because `console.error` is meant to log errors